src/newSeed/seedConfig.ts

```diff
@@ -1,31 +1,34 @@
 export interface ISeedConfig {
-  general: {
+  stage1: {
```

Comment:
please don't refer to stages here.  this json is for public consumption as seed metadata.  It should only refer to seeds, not to the application.